shared/message/output/Buzzer.proto

```diff
@@ -2,4 +2,6 @@ syntax = "proto3";
 
 package message.output;
 
-message Buzzer {}
+message Buzzer {
+    int32 buzzer_frequency = 1;
+}
```

Comment:
Put a comment on the buzzer message. It isn't clear what is buzzing to someone who isn't that familiar with this kind of stuff.

Reply:
Fixed